data/mods/Aftershock/crafting_system.md

```diff
@@ -0,0 +1,67 @@
+# Scrap Crafting System
+
+## Overview
+
+As one of its main  goals, Aftershock allows the crafting of miscellaneous, highly technological devices that have no actual basis in reality.  To make the crafting of these devices more balanced an intuitive, and to make adding more recipes and items easier, aftershock utilizes a crafting a system in which abstracted pieces of increasingly rare scrap are used to craft increasingly useful tools.
+
+
+### Current Scrap Categories
+
+The following table lists the currently planned/implemented scrap categories and their tiers. 
+
+|     Category      |     Tier 1           |    Tier 2                |     Tier 3                 |     Tier 4                |     Tier 5                           |
+| ----------------- | -------------------- | ------------------------ | -------------------------- | ------------------------- | ------------------------------------ |
+| `Circuitry`       | scrap photonics      | photonic circuitry       | photonic computation core  | hypergeometric photonics  | acausal logic permutator             |
+| `Energy Storage`  | Nanowire battery     | ultracapacitor           | Ultracapacitor array       | Superconductive Coil      | zero-point energy extractor          |
+| `Material`        | Composite Superalloy | vacuum cast Carbide      | Nanoprinted Alloys         | crystal forged neutrite   | phase-uneven matter                  |
+| `Cloth`           | E-textile            |                          | graphene weave             | woven metamaterial        |                                      |
+| `Magnet`          | Emag                 | cryo electromagnet       |                            |                           |                                      |
+| `Optics  `        |                      | Laser Optics             | Phased Array Optics        | Nano-optics               |                                      |
+| `Neural I/O`      | peripheral electrode | neural electrode         | Brain implant prod         | Synthetic Neural Tissue   | neurosynaptic interface matrix       |
+| `Biomaterial`     | monomeric slurry     | micellular growth medium | artificial muscle fibers   | self healing polymers     | autologous totipotent tissue culture |
+
+Note that more categories can be added, and that it inst necessary for a category to encompass the five tiers, although its better if they do.
+
+## Scrap Tiers Briefly Explained
+
+### Tier 1
+
+Tier 1 scarp is plentiful, and can be easily found even when you aren't specifically looking for it.  It might be dropped by defeating common enemies, or by disassembling common household objects.  Ideally Tier 1 scarp is mostly used as a filler component in more advanced recipes. Things that might be crafted using only this tier of scrap must be single-use and not very effective, and should  see little use outside of the early game.For example
```

Comment:
In this paragraph you've got two typos `scarp` instead of `scrap, also in Tier 2. Those are the only occurrences I believe.